pyproject.toml

```diff
@@ -1,23 +1,182 @@
 [build-system]
-requires = ["setuptools>=46.1.0", "setuptools_scm[toml]>=5", "wheel"]
+requires = ["setuptools>=65.4.1", "setuptools_scm[toml]>=7", "wheel"]
 build-backend = "setuptools.build_meta"
 
+
+[project]
+name = "iqm-cortex_cli"
+dynamic = ["version"]
+description = "CLI for executing quantum circuits"
+authors = [{name = "IQM Finland Oy", email = "developers@meetiqm.com"}]
+readme = "README.rst"
+license = {file = "LICENSE.rst"}
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Programming Language :: Python :: 3 :: Only",
+    "Topic :: Scientific/Engineering :: Physics",
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: Apache Software License"
+]
+requires-python = ">=3.9, <3.11"
+dependencies = [
+    "click >= 7.1.2",
+    "jsonschema >= 4.6.0",
+    "psutil >= 5.9.2",
+    "pydantic >= 1.10.2, < 2.0",
+    "python-daemon >= 2.3.0",
+    "requests >= 2.26.0"
+]
+
+[project.urls]
+homepage = "https://github.com/iqm-finland/cortex-cli"
+documentation = "https://iqm-finland.github.io/cortex-cli"
+repository = "https://github.com/iqm-finland/cortex-cli.git"
+changelog = "https://github.com/iqm-finland/cortex-cli/blob/main/CHANGELOG.rst"
+
+[project.optional-dependencies]
+# Add here additional requirements for extra features, to install with:
+# `pip install iqm-cortex-cli[testing,docs,dev]`
+circuit = [
+    "cirq-iqm >= 11.0, < 12.0",
+    "iqm-client >= 11.0, < 12.0"
+]
+dev = [
+    "tox == 3.25.1"
+]
+testing = [
+    "black == 22.10.0",
+    "isort == 5.10.1",
+    "mypy == 0.991",
+    "pytest == 7.1.2",
+    "pytest-cov == 3.0.0",
+    "pytest-isort == 3.0.0",
+    "pytest-mypy == 0.9.1",
+    "pytest-pylint == 0.18.0",
+    "mockito == 1.2.2",
+    "types-requests == 2.28.9"
+]
+docs = [
+    "sphinx == 4.5.0",
+    "sphinx-book-theme == 0.3.3"
+]
+cicd = [
+    "twine ~= 4.0"
+]
+
+[project.scripts]
+cortex = "cortex_cli.cortex_cli:cortex_cli"
+
+
+[tool.setuptools]
+platforms = ["any"]
+zip-safe = false
+include-package-data = true
+package-dir = {"" = "src"}
+
+[tool.setuptools.packages.find]
+where = ["src"]
+exclude = ["tests"]
+
```

Comment:
```suggestion
```
I think all of these are unnecessary, the default setuptools file discovery method should work fine with our project setup.